crates/aiken-lang/src/tests/check.rs

```diff
@@ -1849,7 +1849,10 @@ fn forbid_expect_into_opaque_type_constructor_without_typecasting_in_module() {
         }
     "#;
 
-    assert!(check(parse(source_code)).is_ok());
+    assert!(matches!(
```

Comment:
This was straight up the wrong assert somehow, we should not have been checking `is_ok` here. And so this wasn't working correctly funny enough.

Reply:
Well no, this is actually okay. Because the constructor is in the same module.

Reply:
No we said not even in the same module could you use expect. Cause you should just use when!